README.md

```diff
@@ -42,29 +40,42 @@ If a user clicks on the `Pay` button, a dialog detailed [here](./design-spec/pay
 Upon confirmation, a call should be made to the API that updates the state of that payment to `Paid`.  
 The result of that call should be used to then update the status of that transaction in the list that is displayed to the user.
 
-For the left-hand navigation. The other two menu items do not have to do anything in particular. But they should be distinguishable from one-another (e.g. a simple component that just displays the name of the menu item is fine). They are merely there for you to demonstrate navigation between content using React.
+**To get started, make changes to `~/src/pages/index.ts`.**
 
-**NB:**
-You are not expected to match any of the styles, colour, fonts etc in the given design images exactly. These are just guidelines for how we would roughly like the site to be laid out and look.
+A basic index page has been fleshed out for you that fetches raw payment data from the API - this is just an example and does not indicate any code, patterns, styles, or naming convetions that we expect you to follow or use, feel free to make changes as you see fit.
+
+Please make commits/PRs to this repository like you would at work.
+
+> **NOTE:**  
+> The frontend repository was created using Next.js.  
+> This is because it
+>
+> - Is a convenient framework to build a React web app with a self-contained API.
+> - Something we use here at Landlord Studio.
+>
```

Comment:
This could probably be more concise, something like
```
This project was created using Next.js, a tool we use here at Landlord Studio. It provides tools to build a React web app with a self-contained API.
You shouldn't need to use or know anything Next.js specific to complete this test, if you want/need to know more however, documentation can be found [here](https://nextjs.org/docs).
```

Reply:
Yeh that is better, amended